config/docker/php/Dockerfile

```diff
@@ -1,11 +1,43 @@
-FROM 4xxi/php-pgsql:flex
+FROM php:7.2-fpm-alpine
 
-# To add additional packages and php extensions uncomment the following lines and add required dependencies
-# Check https://hub.docker.com/_/php/ for examples and documentation
+COPY ./symfony.ini /usr/local/etc/php/conf.d/
+COPY ./symfony.pool.conf /usr/local/etc/php-fpm.d/
```

Comment:
If symfony.ini of symfony.pool.conf will be changed whole image must be rebuilded. Better make this operations at the end of Dockerfile, so rebuild will affect only latest layers.